protocol/x/vault/keeper/orders.go

```diff
@@ -10,3 +33,130 @@ import (
 // 2. Placing new orders.
 func (k Keeper) RefreshAllVaultOrders(ctx sdk.Context) {
 }
+
+// GetVaultClobOrders returns a list of long term orders for a given vault, with its corresponding
+// clob pair, perpetual, market parameter, and market price.
+// Let n be number of layers, then the function returns orders at [a_1, b_1, a_2, b_2, ..., a_n, b_n]
+// where a_i and b_i are the ask price and bid price at i-th layer. To compute a_i and b_i:
+// - a_i = oraclePrice * (1 + spread)^i
+// - b_i = oraclePrice * (1 - spread)^i
+// TODO (TRA-144): Implement order size
+// TODO (TRA-114): Implement skew
+func (k Keeper) GetVaultClobOrders(
+	ctx sdk.Context,
+	vaultId types.VaultId,
+) (orders []clobtypes.Order) {
```

Comment:
minor nit: non-blocking, do you expect the orders returned to be passed around into other functions / helpers alot? Could make sense to return a slice of pointers to order structs.

Reply:
Good catch! A slice of pointers will be more efficient